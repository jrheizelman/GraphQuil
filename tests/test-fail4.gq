/* 
 * Author: Gemma Ragozzine
 * Tests wrong types
 */

int main () {
	String t;
	1 + t;
}