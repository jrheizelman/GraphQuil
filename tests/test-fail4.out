Fatal error: exception Failure("Operator + not compatible with expressions of type int and  String.")